source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -195,14 +194,7 @@ content: |
            connection_string = "mongodb://localhost:27017"
            key_vault_namespace = "encryption.__keyVault"
 
-           fle_opts = AutoEncryptionOpts(
-               kms_providers,  # pass in the kms_providers variable from  the previous step
-               key_vault_namespace
-           )
-           client = MongoClient(
-               connection_string,
-               auto_encryption_opts=fle_opts
-           )
+           client = MongoClient(connection_string)
```

Comment:
The client passed into `ClientEncryption` does not need to have auto_encryption enabled.